lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -325,15 +317,6 @@ def get_axis_position(self):
         zhigh = tc[0][2] > tc[2][2]
         return xhigh, yhigh, zhigh
 
-    def _on_units_changed(self, scalex=False, scaley=False, scalez=False):
-        """
-        Callback for processing changes to axis units.
-
-        Currently forces updates of data limits and view limits.
-        """
-        self.relim()
-        self.autoscale_view(scalex=scalex, scaley=scaley, scalez=scalez)
```

Comment:
And this one is `autoscale_view`, not `_request_autoscale_view`. I don't know if that's relevant.

Reply:
It doesn't really matter (`_request_autoscale_view` just delays the autoscale until someone accesses them) but I also reverted that change given the problems above anyways).